txp/Pos/Txp/Logic/Local.hs

```diff
@@ -54,7 +56,8 @@ txProcessTransaction
 txProcessTransaction itw@(txId, txAux) = do
     let UnsafeTx {..} = taTx txAux
     tipDB <- GS.getTip
-    bootEra <- gsIsBootstrapEra
+    epoch <- siEpoch <$> getCurrentSlotBlocking
```

Comment:
No, it's totally not ok. If we receive multiple transactions, we will have multiple threads waiting for slotting data.
If we don't know current slot, we should reject transactions.

Btw, Misha's PR (for CSL-1205) was so big that I didn't notice that it was very bad in this part.
1. He added this logic to `txProcessTransaction`, but didn't add it to explorer version.
2. He also didn't add these checks to block verification.
3. He didn't take this logic into account in normalization. So now after normalization we can have invalid transactions in mempool :(
3. In fact, it should be in Toil, not in Logic, in this case all three problems above would be resolved.

And in fact the task is more complicated.
After thinking about it for a while, I decided that it doesn't make sense to fix it within this issue. I will make a separate one.
As a workaround please use `getCurrentSlotInaccurate`.